OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -3110,6 +3119,15 @@ public static void sendOutcome(@NonNull String name, OutcomeCallback callback) {
       outcomeEventsController.sendOutcomeEvent(name, callback);
    }
 
+   static void sendUniqueClickActionOutcomeEvent(@NonNull String name) {
```

Comment:
actually i need this, is for differentiate a common outcome from an IAM outcome, for the time being we are making all unattributed. If you look on the OutcomeVentsController there is sendUniqueClickOutcomeEvent and sendClickOutcomeEventWithValue that work with the getIAMSessionResult().

After all IAM stuff is made this will be remove. Do you want me to put a comment regarding this?